cli/.gitignore

```diff
@@ -0,0 +1,17 @@
+
```

Comment:
Falls diese Sachen noch nicht abgedeckt werden von der globalen gitignore kann das glaube ich alles in die globale.
Gerade .exe und co wollen wir ja nie commiten.

Reply:
Würde es Projekt/Dirspezifisch lassen, damit es klar strukturiert ist. :-)